### PR TITLE
Configure rollbar js

### PR DESCRIPTION
**Pull request by amiedes**
Closes [#623](https://github.com/PopulateTools/issues/issues/623)

## :v: What does this PR do?

Enables Rollbar for reporting JS errors.

## :shipit: Does this PR changes any configuration file?

- [x] new environment variable in `.env.example`?
- [ ] new entry in `config/application.yml`?
- [x] new entry in `config/secrets.yml`?

(Changes in these files might need to update the role in Ansible)

### CODE REVIEW

File: config/initializers/rollbar.rb [hunk 1]
```diff
@@ -4,10 +4,14 @@
 
 Rollbar.configure do |config|
   config.access_token = Rails.application.secrets.rollbar_access_token
-
   config.enabled = false if Rails.env.development? || Rails.env.test?
-
   config.exception_level_filters.merge!("ActionController::InvalidCrossOriginRequest" => "ignore",
                                         "ActionController::RoutingError" => "ignore",
                                         "ActionController::UnknownFormat" => "ignore")
+  config.js_enabled = true
```
Comment (ferblape) [inline config/initializers/rollbar.rb]: It should be only enabled in staging or production, don't you think?

Comment (amiedes) [reply config/initializers/rollbar.rb]: Right. I've fixed it.

**Comment by ferblape** [comment]
Is the ansible role ready? I can't see any reference of the new api key configured in the server.

**Comment by amiedes** [comment]
> Is the ansible role ready? I can't see any reference of the new api key configured in the server.

Yes, but it hadn't been executed on production yet, I just did it.

**Comment by ferblape** [comment]
Ok, thanks. I'm going to merge this, Rollbar only delays 2ms the load of the page, it's working pretty fine, and we have found a few errors, so it's helpful too.

**Pull request merged**